src/options/options.css

```diff
@@ -1,7 +1,7 @@
 body {
-  /* the style the other options in the settings pane use */
-  font-size: 1.11em;
   color: #333;
+  font-family: sans-serif;
```

Comment:
Okay, on Windows this results in `Arial` and `Tahoma` instead of `Segoe UI` and `Tahoma`.

## Arial (with `sans-serif`)

![grafik](https://user-images.githubusercontent.com/11966684/54416741-8a74e700-4700-11e9-9ab2-dff99c6aeff5.png)

## Segoe UI (without `sans-serif`)

![grafik](https://user-images.githubusercontent.com/11966684/54416683-64e7dd80-4700-11e9-93fd-47d4eaaf49ec.png)

Note that we use a basic `browser-style` for [the optiosn page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Options_pages) (see at the bottom), so AFAIK the best font should be provided by Firefox itself. See also https://stackoverflow.com/questions/39830037/web-extension-how-do-i-use-browser-style-true

ANd IMHO the "Segoe UI" version also looks closer to the original Firefox settings page than Arial.

Reply:
the `browser-style` uses the browser's default font setting instead. So for example, if my default font is a mono-font, then it will use the mono-font. This is different from the font `about:preference` uses.

So on my own setting, i get a mono font for the option page, but a sans-serif font for about:preference

Reply:
but either ways, i can remove this, so it will use the user's default font settings.